code/modules/mob/new_player/new_player.dm

```diff
@@ -549,8 +482,9 @@
 /mob/new_player/handle_reading_literacy(var/mob/user, var/text_content, var/skip_delays)
 	. = text_content
 
-/mob/new_player/handle_writing_literacy(var/mob/user, var/text_content, var/skip_delays)
-	. = text_content
```

Comment:
Why did this get removed?

Reply:
See above its the same proc just copy pasted

Reply:
Because it's an override. Put it back please.

Reply:
I put it back.